extensions/amp-story/1.0/amp-story.js

```diff
@@ -706,6 +715,14 @@ export class AmpStory extends AMP.BaseElement {
           if (infoDialog) {
             infoDialog.build();
           }
+        })
+        .then(() => {
+          this.hasBookend_().then(hasBookend => {
+            if (hasBookend && bookendActive) {
+              return this.showBookend_();
```

Comment:
Does this block story layout in the event that the bookend XHR fails?  Can we add a test to ensure that doesn't happen?